stdlib/2/tokenize.pyi

```diff
@@ -102,15 +103,15 @@ chain = ...  # type: type
 double3prog = ...  # type: type
 endprogs = ...  # type: Dict[str, Any]
 pseudoprog = ...  # type: type
-re = ...  # type: module
```

Comment:
Honestly a lot of here should be removed -- it's only there because that's how pytype generates stubs (every imported thing ends up in the stub, even if it's clearly for purely internal reasons, and it never generates import statements). But I think that's a cleanup for a separate PR.